go/vt/sqlparser/sql.y

```diff
@@ -1298,7 +1305,7 @@ alter_statement:
   {
     $$ = &DDL{Action: AlterStr, Table: $4}
   }
-| ALTER ignore_opt TABLE table_name RENAME column_opt ID to_opt ID
+| ALTER ignore_opt TABLE table_name RENAME COLUMN ID TO ID
```

Comment:
Tighten this up to follow MySQL's actual grammar.

Reply:
I'll be damned, you're right